src/script/components/list/GroupedConversationHeader.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Wire
+ * Copyright (C) 2019 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import React from 'react';
+import cx from 'classnames';
+
+import {registerReactComponent} from 'Util/ComponentUtil';
+
+import type {ConversationLabel} from '../../conversation/ConversationLabelRepository';
+import NamedIcon from '../NamedIcon';
+
+export interface GroupedConversationHeaderProps {
+  conversationLabel: ConversationLabel;
+  isOpen: boolean;
+}
+
+const GroupedConversationHeader: React.FC<GroupedConversationHeaderProps> = ({conversationLabel, isOpen}) => {
+  const badge = conversationLabel.conversations().filter(conversation => conversation.hasUnread()).length;
```

Comment:
That might not work if just the `conversations` observable is changing but `conversationLabel` and `isOpen` stay the same. Would that be the case in reality or is `conversationLabel` always changing together with the containing `conversations`?

Reply:
Resolved in 1:1